lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2095,6 +2095,50 @@ void SwiftLanguageRuntime::DidFinishExecutingUserExpression(
 
 bool SwiftLanguageRuntime::IsABIStable() { FORWARD(IsABIStable); }
 
+struct AsyncUnwindRegisters {
+  uint32_t async_ctx_regnum;
+  uint32_t fp_regnum;
+  uint32_t pc_regnum;
+  uint32_t dummy_regnum;
+};
+
+static llvm::Optional<AsyncUnwindRegisters>
+GetAsyncUnwindRegisters(llvm::Triple::ArchType triple) {
+  switch (triple) {
+  case llvm::Triple::x86_64:
+    return (AsyncUnwindRegisters){
+        .async_ctx_regnum = dwarf_r14_x86_64,
+        .fp_regnum = dwarf_rbp_x86_64,
+        .pc_regnum = dwarf_rip_x86_64,
+        .dummy_regnum = dwarf_r15_x86_64,
```

Comment:
It would be nice if we could get this definition from header in LLVM, but I guess there probably is none that would define the DWARF regnum for this, right? But it would be nice if this were defined in some tablegen-erated header in LLVM instead.

Reply:
to be fair, we use these constants in a half dozen places in lldb already.